CMakeLists.txt

```diff
@@ -40,6 +40,36 @@ if (mmtf_build_examples)
     add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/examples)
 endif()
 
+
+if (build_py)
+    if(NOT CMAKE_BUILD_TYPE)
+      set(CMAKE_BUILD_TYPE Release)
+    endif()
+    set(CMAKE_CXX_FLAGS "-Wall -Wextra")
+    set(CMAKE_CXX_FLAGS_DEBUG "-g")
+    set(CMAKE_CXX_FLAGS_RELEASE "-O3")
+    add_definitions(-DNDEBUG)
+    add_library(mmtf_py SHARED
+    	${CMAKE_CURRENT_SOURCE_DIR}/bindings/bindings.cpp
+    )
+
+    set(MSGPACKC_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/msgpack-c/include)
+    set(PYBIND_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/pybind11/include)
+	# set(PYTHON_INCLUDE_DIR_A python_include_A)
+    # set(PYTHON_INCLUDE_DIR_B python_include_B)
+
+	target_include_directories(mmtf_py PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
```

Comment:
Picky observation: there's a mix of tabs and spaces here. It would be preferred to stick to one style at least within the same file.